administracion/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
This code won't pass our linting process, as TestCase is currently being imported but unused. 

I suggest commenting this line until actual test cases are written.